<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialogg</class>
 <widget class="QDialog" name="Dialogg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>姓名/身份证号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineedit5"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="button7">
           <property name="text">
            <string>确认</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button8">
           <property name="text">
            <string>返回</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
